anchore_engine/analyzers/modules/32_java_packages.py

```diff
@@ -93,22 +91,15 @@ def process_java_archive(prefix, filename, inZFH=None):
                     top_el['metadata']['MANIFEST.MF'] = anchore_engine.utils.ensure_str(MFH.read())
 
                 manifest = java_util.parse_manifest(top_el['metadata']['MANIFEST.MF'].splitlines())
-                sname = manifest['Specification-Title']
-                sversion = manifest['Specification-Version']
-                svendor = manifest['Specification-Vendor']
-                iname = manifest['Implementation-Title']
-                iversion = manifest['Implementation-Version']
-                ivendor = manifest['Implementation-Vendor']
-
-                if sversion:
-                    top_el['specification-version'] = sversion
-                if iversion:
-                    top_el['implementation-version'] = iversion
-
-                if svendor:
-                    top_el['origin'] = svendor
-                elif ivendor:
-                    top_el['origin'] = ivendor
+                if 'Specification-Version' in manifest:
```

Comment:
I think this can be simplified as:

top_el['specification-version'] = manifest.get('Specification-Version', 'N/A')
top_el['implementation-version'] = manifest.get('Implementation-Version', 'N/A')
...

Rather than having to use if-statements for each.

Reply:
Good catch. That aligns with how its done in the maven section too. I'll update.